src/Integrations/Mailchimp/SettingsMailchimp.php

```diff
@@ -264,7 +266,7 @@ public function getSettingsGlobalData(): array
 						'checkboxId' => $this->getSettingsName(self::SETTINGS_MAILCHIMP_USE_KEY),
 						'checkboxLabel' => __('Use Mailchimp', 'eightshift-forms'),
 						'checkboxIsRequired' => true,
-						'checkboxIsChecked' => !empty($this->getOptionValue(self::SETTINGS_MAILCHIMP_USE_KEY)),
+						'checkboxIsChecked' => $this->getOptionValue(self::SETTINGS_MAILCHIMP_USE_KEY) === 'true',
```

Comment:
Why is this a string?

Reply:
I'd recon that the `getOptionValue` method returns a string. I'd guess it can return more than just true/false values. Like keys and other things.

Reply:
if the checkbox value is set it will be used if the checkbox is checked. If it is not checked it will return false. I'm trying to reduce type jugling here so I cast all in string. Because bool can be set in the db as true and returned as true but outputed as 1 and so on. In this way sa string I'm safe (ish)